src/app/baseline/wizard/capability/capability.component.ts

```diff
@@ -100,69 +106,51 @@ export class CapabilityComponent implements OnInit {
         this.baselineObjectAssessments = baselineObjectAssessments;
       }, (err) => console.log(err));
 
-    const sub3$ = this.wizardStore
+    this.wizardStore
       .select('baseline').pipe(
       pluck('currentObjectAssessment'))
-      // .distinctUntilChanged()
       .subscribe(
         (currentObjectAssessment: ObjectAssessment) => {
           this.currentObjectAssessment = currentObjectAssessment;
-          if (this.currentObjectAssessment) {
```

Comment:
Trying to figure out what needs to be added back in....... I had the logic setup like this so that when you changed a pdr score the page woudn't jump to the bottom.